### DISCUSSION TITLE
Lottie File on Iconsout

### DISCUSSION THREAD

**Discussion by champolot** [Kyutefox/Iconify#22] [Ideas]
Would it be possible ot have an abiltity to download lottie files in icounsout?

**Comment by MrJukeman** [comment]
@champolot Certainly! IconScout has recently implemented changes on their website, and we are currently working on fixing any issues that may have arisen due to these updates. We'll ensure that the ability to download Lottie files on IconScout is restored as soon as possible.

**Comment by MrJukeman** [comment]
@champolot The update has been made. Enjoy you lottie Icon ❤️

**Reply by champolot** [reply]
Cool! Thanks for the updates! Highly appreciated Razoo!
